src/icons/CatalogIcon/CatalogIcon.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC } from 'react';
+import { DEFAULT_HEIGHT, DEFAULT_WIDTH } from '../../constants/constants';
+import { CARIBBEAN_GREEN, CHARCOAL, KEPPEL, LIGHT_GRAY, useTheme } from '../../theme';
+import { IconProps } from '../types';
+
+type CatalogIconProps = {
+  primaryFill?: string;
+  secondaryFill?: string;
+} & IconProps;
+
+export const CatalogIcon: FC<CatalogIconProps> = ({
+  width = DEFAULT_WIDTH,
+  height = DEFAULT_HEIGHT,
+  primaryFill,
+  secondaryFill,
+  style = {},
+  ...props
+}) => {
+  const theme = useTheme();
+
+  const themePrimaryFill = primaryFill ?? (theme.palette.mode === 'dark' ? LIGHT_GRAY : KEPPEL);
```

Comment:
@nebula-aac is this representative of our approach to light and dark mode support?